womtool/src/main/scala/womtool/WomtoolMain.scala

```diff
@@ -75,25 +75,44 @@ object WomtoolMain extends App {
   }
 
   def upgrade(workflowSourcePath: String): Termination = {
```

Comment:
ToL (future ticket idea) I'm almost certain that a `-i` flag for "upgrade in place" is going to be a big request for people who have multiple interdependent WDL draft-2 files to upgrade

Reply:
ToL again: this is probably what you'll end up having to implement *somewhere* to make the centaur tests work?